engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/selectModulesScreen/SelectModulesScreen.java

```diff
@@ -551,4 +597,32 @@ private void refreshSelection() {
         setSelectedVersions(resolver.resolve(selectedModules));
         updateValidToSelect();
     }
+
+    // TODO: should dynamically generate checkbox list from boolean StandardModuleExtensions rather than hardcoding associations here
```

Comment:
Should dynamically generate the contents of the checkbox list rather than hoping every Boolean StandardModuleExtension is listed with a matching checkbox id.   I might consider doing that in a future PR.

Reply:
Could be a nice Good First Issue with a nice little write-up after this is merged 👍